packages/eslint-plugin-material-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-material-ui",
-  "version": "6.0.0",
+  "version": "6.0.4",
```

Comment:
Nitpick: Was the bump "suggested" by lerna or did you do it manually? 🤔

Reply:
Suggested, but I don't know why

Reply:
Gotcha. Could be because of an `eslint` package bump. 💡 🤔